src/usdb_syncer/download_options.py

```diff
@@ -6,6 +6,13 @@
 from usdb_syncer import settings
 
 
+@dataclass(frozen=True)
+class MiscOptions:
```

Comment:
Unfortunately this cannot be added to `TxtOptions` since this might not be presented when "Download Song file" is unchecked. This option always has to be present since it's also used to sanitize the paths made up of artist & title.
I didn't have a good idea for a class name.